test-utils/runtime-tester/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+/// run_test is a framework for creating and executing runtime scenarios.
+/// You can create Scenario in rust code or have it in a JSON file.
+/// Scenario::run executes scenario, keeping track of different metrics.
+/// So far, the only metric is how much time block production takes.
```

Comment:
```suggestion
//! run_test is a framework for creating and executing runtime scenarios.
//! You can create Scenario in rust code or have it in a JSON file.
//! Scenario::run executes scenario, keeping track of different metrics.
//! So far, the only metric is how much time block production takes.
```

With `///`a this would be a doc-comment for the following item, the `run_test` module. With `//!`, this will be a comment for the current item, the whole crate.